Jenkinsfile

```diff
@@ -2,20 +2,49 @@
 found at https://github.com/ihmeuw/vivarium_build_utils
 Due to Jenkins convention, importable modules must be stored
 in the 'vars' folder.
-Jenkins needs to be configured globally to use the correct branch.
```

Comment:
We are no longer using the `@Library` directive in favor of `library` directive because @Library is run at compile time and so can't support a dynamic value like the version number.